test-dts/defineComponent.test-d.tsx

```diff
@@ -905,13 +911,18 @@ describe('emits', () => {
       expectError(this.$emit('input'))
       //  @ts-expect-error
       expectError(this.$emit('input', 1))
+    },
+    render() {
+      // this.$props.
     }
   })
 
   // with array emits
   defineComponent({
     emits: ['foo', 'bar'],
     setup(props, { emit }) {
+      expectType<any>(props.onFoo)
```

Comment:
This should rather be something like the onClick:

```suggestion
       expectType<(((...args: any[]) => any) | undefined)>(props.onFoo)
```

and same for onBar